SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -73,14 +73,9 @@ public async Task<IActionResult> CreateArticleAsync([FromBody] CreateArticleDTO
                 return ValidationProblem(response);
             }
 
-            bool createdSuccessful = await _articleService.CreateArticleAsync(adminInput);
+            string createdSuccessful = await _articleService.CreateArticleAsync(adminInput);
 
-            if(createdSuccessful)
-            {
-                return Ok(ValidationMessages.ArticleCreatedSuccessfully);
-            }
-
-            return BadRequest(ValidationMessages.UnableToCreateArticle);
+            return Ok(createdSuccessful);
```

Comment:
Please, return here Created(string.empty, createdArticle). I think for now it is the best case

Reply:
Okay, I changed it.